tools/policy-gen/templates/schema.yaml

```diff
@@ -11,4 +11,7 @@ properties:
   name:
     description: 'Name of the Kuma resource'
     type: string
+  labels:
```

Comment:
it's probably a good idea to add
```yaml
additionalProperties:
  type: string
```
to limit key/value of the object to be `string`. Also, that's how k8s defines `labels` https://github.com/kubernetes/kubernetes/blob/master/api/openapi-spec/v3/apis__policy__v1_openapi.json#L876-L883